cypress/support/commands.js

```diff
@@ -25,17 +25,52 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
+import faker from 'faker';
+import { generateNewArticle } from './generateNewArticle';
+import { generateNewUser } from './generateNewUser';
 
 addMatchImageSnapshotCommand();
 
+Cypress.Commands.add(`findByPlaceholder`, (placeholder) => {
+  cy.get(`[placeholder = "${placeholder}"]`);
+});
+
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[data-qa="${selector}"]`);
```

Comment:
The 'getByDataCy' command does not return a value. Add a 'return' statement to return the Cypress command for chaining.